### PR TITLE
Move browser data to risk metadata

### PR DESCRIPTION
**Pull request by ashkarpetin**

### CODE REVIEW

File: openapi/components/schemas/RiskMetadata.yaml [hunk 1]
```diff
@@ -1,4 +1,6 @@
 type: object
+title: Browser data
```
Comment (ashkarpetin) [inline openapi/components/schemas/RiskMetadata.yaml:2]:
```suggestion
title: Risk metadata
```

**Review by VladimirBerdnik — DISMISSED** [review]
LGTM

**Review by iamdevice — DISMISSED** [review]

**Review by XaosSintez — DISMISSED** [review]

**Review by slavcodev — DISMISSED** [review]

File: openapi/components/schemas/RiskMetadata.yaml [hunk 2]
```diff
@@ -5,11 +5,13 @@ properties:
   ipAddress:
     description: The customer's IP.
     type: string
-    format: ipv4
+    format: ipv4 or ipv6
```
Comment (slavcodev) [inline openapi/components/schemas/RiskMetadata.yaml:8]: This is quite weird format, I'm 99% sure that there is no any tools which will understand it. To make it works, consider this schema
```json
ipAddress:
  description: The customer's IP.
  type: string
  oneOf:
    - format: ipv4
      example: 93.92.91.90
    - format: ipv6
      example: 2354:0425:2CA5:0000:0000:0657:5673:23b6
```

Comment (slavcodev) [reply openapi/components/schemas/RiskMetadata.yaml:8]: however I\m not sure Redoc supports such json-schema

**Review by slavcodev — DISMISSED** [review]

File: openapi/components/schemas/RiskMetadata.yaml (hunk 2 above)
Comment (slavcodev) [reply openapi/components/schemas/RiskMetadata.yaml:8]: anyway, I approved the PR in case you'll dismiss this suggestion.

Comment (ashkarpetin) [reply openapi/components/schemas/RiskMetadata.yaml:8]: It was already discussed [here](https://github.com/Rebilly/api-definitions/pull/33/commits/1772172b49212b3b6a00849e291622b6b1d5b278#r416491569). 


> The OpenAPI specification only supports these formats:
> 
> https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.3.md#dataTypeFormat
> 
> Other formats are valid.
> 
> However, you cannot expect tooling even to support those pre-defined formats, let alone any other format.

Comment (ashkarpetin) [reply openapi/components/schemas/RiskMetadata.yaml]: @adamaltman 

Which one is better?

`format: ipv4 or ipv6`
scr

`oneOf`
scr2

Comment (slavcodev) [reply openapi/components/schemas/RiskMetadata.yaml]: Ok I see. We write OAS definition for docs (Redoc), and does not consider the other tooling:  validation, etc.
Please disregard this suggestion.

**Review by iamdevice — DISMISSED** [review]

File: openapi/components/schemas/RiskMetadata.yaml [hunk 3]
```diff
@@ -1,9 +1,32 @@
 type: object
+title: Risk metadata
+description: Risk metadata used for 3DS and risk scoring.
 properties:
   ipAddress:
-    description: The customer's IP.
+    description: The customer IP address (ipv4 or ipv6).
     type: string
-    format: ipv4
+    oneOf:
```
Comment (adamaltman) [inline openapi/components/schemas/RiskMetadata.yaml:8]: This suggestion was incorrect. First it should be anyOf, not oneOf. Second, I think this is overcomplicated and adds no value.

**Review by adamaltman — APPROVED** [review]

**Review by iamdevice — APPROVED** [review]

**Review by creocoder — APPROVED** [review]

**Review by slavcodev — APPROVED** [review]

**Pull request merged**